### PR TITLE
Dev om

### PR DESCRIPTION
**Pull request by omiossec**
## Description
Add a new function Get-WebServerInfo to retrieve server information from HTTP Respons header

## Related Issue
Part of #1

## Motivation and Context
Sometimes you need to retrieve server information from a website (for example a website) it's easier if we can do it directly while discussing in Slack or Team

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x ] My code follows the code style of this project.
- [ x] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x ] I have read the **CONTRIBUTING** document.
- [ x] I have added tests to cover my changes.
- [x ] All new and existing tests passed.

### CODE REVIEW

File: PoshBot.Networking/PoshBot.Networking.psm1 [hunk 1]
```diff
@@ -106,5 +106,72 @@ function Invoke-TestPort {
...
 
+function get-webServerInfos {
+    <#
+    .SYNOPSIS
+        Check HTTP header for serverinfo
+    .EXAMPLE
+        !webserverinfo http://uri
+    #>
+    [PoshBot.BotCommand(
+        CommandName = 'testport',
```
Comment (devblackops) [inline PoshBot.Networking/PoshBot.Networking.psm1]: Copy paste error? This should be something like `webserverinfo`.

File: PoshBot.Networking/PoshBot.Networking.psm1 [hunk 2]
```diff
@@ -106,5 +106,72 @@ function Invoke-TestPort {
...
+    }
+    catch {
+        write-debug $_.Exception
+        $ErrorMessage =  "Receive a general error " +  $_.Exception
+    }
+
+    finally {
+
+        if ($ErrorMessage) {
+            New-PoshBotCardResponse -Type Warning -Text "$ErrorMessage :(" -Title 'Rut row' -ThumbnailUrl 'http://images4.fanpop.com/image/photos/17000000/Scooby-Doo-Where-Are-You-The-Original-Intro-scooby-doo-17020515-500-375.jpg'
```
Comment (devblackops) [inline PoshBot.Networking/PoshBot.Networking.psm1]: I have the thumbnails used by the builtin plugin hosted in the main PoshBot repo. You can use this link so we're not relying on the other site.

`https://raw.githubusercontent.com/poshbotio/PoshBot/master/Media/scooby_doo.jpg`

Comment (omiossec) [reply PoshBot.Networking/PoshBot.Networking.psm1]: Ok I will change the 2 points

**Comment by devblackops** [comment]
Thanks @omiossec! Looks like the other PRs were duplicated in this one I'm going to close those out.

**Pull request merged**